src/main/webapp/go.js

```diff
@@ -436,6 +445,44 @@ function checkUserDestinationGuess() { //eslint-disable-line
   });
 }
 
+/**
+ * Retrieves the place ID of the current destination, retrieved
+ * using the Places library.
+ */
+function getPlaceIDOfAnswer() {
+  const answerRequest = {
+    query: hunt.getCurDestName(),
+    fields: ['place_id'],
+  };
+  service.findPlaceFromQuery(answerRequest, function(results, status) {
+    if (status === google.maps.places.PlacesServiceStatus.OK) {
+      return results[0].place_id;
+    } else {
+      return '';
+    }
+  });
+}
+
+/**
+ * Retrieves the place ID of the user's guess, retrieved
+ * using the Places library.
+ * @param {String} userGuess The user's guess for the name of the
+ * current destination.
+ */
+function getPlaceIDOfGuess(userGuess) {
```

Comment:
I think this should also be changed to use a single implementation of the common code referred to in my comment above.

Reply:
@megankj 

I decided to nest the second call to findPlaceFromQuery inside the first call -- I put some of the duplicated code into a common function (createRequestForPlaceID), but I'm not sure how to combine the rest of the redundant code.